2-1-4.py

```diff
@@ -0,0 +1,3 @@
+#print(input().count(' ') + 1) - don't work =)
```

Comment:
What is this for? Usually it's better not to leave commented code lines - they just add noise.

Reply:
I kick this comment! =)